### DISCUSSION TITLE
Student pack issue

### DISCUSSION THREAD

**Discussion by MatyAlts** [community/community#124002] [GitHub Education]
### Hi everyone,

Hi, i finished the Intro to GitHub Flow, but my student pack isn't activated

**Comment by Rishit-katiyar** [comment]
Hey there,

Congrats on completing the Intro to GitHub Flow! It's great to hear you're diving into the platform. Now, about your student pack not being activated, let's see what we can do:

1. **Double-check your eligibility**: Make sure you meet all the requirements for the GitHub Student Developer Pack. You should be a student enrolled in a degree or diploma-granting course and at least 13 years old.

2. **Verify your email**: Ensure the email address linked to your GitHub account is the same one you used for your student pack application. GitHub usually confirms your student status through this email.

3. **Get in touch with GitHub Support**: If you're eligible and your email is verified but you still haven't got your student pack, reach out to GitHub Support for help. They're usually quick to respond and can guide you through the process.

   Here's how you can contact them:
   - Go to the GitHub Education Support form [GitHub Education Support](https://support.github.com/contact/education).
   - Fill out the form with your issue details and any relevant documents.
   - GitHub Support will review your case and assist you further in activating your student pack.

4. **Be patient**: Sometimes, it takes a bit of time for everything to get sorted out, so keep an eye on your email for updates from GitHub.
